src/components/controls/controls.js

```diff
@@ -0,0 +1,222 @@
+var registerControls = require('../../core/controls').registerControls;
+var THREE = require('../../lib/three');
+
+var MAX_DELTA = 0.2; // ms
+var PI_2 = Math.PI / 2;
+
+/**
+ * Controls component.
+ *
+ * Receives input events from device-specific control components, and applies movement and rotation
+ * to the element accordingly.
+ */
+module.exports.Component = registerControls('controls', {
+
+  dependencies: ['velocity', 'rotation'],
+
+  schema: {
+    enabled: { default: true },
+
+    flyingEnabled: { default: false },
+
+    position: { default: ['gamepad-controls', 'keyboard-controls', 'touch-controls'] },
+    positionEnabled: { default: true },
+    positionEasing: { default: 15 }, // m/s2
+    positionAcceleration: { default: 65 }, // m/s2
+
+    rotation: { default: ['hmd-controls', 'gamepad-controls', 'mouse-controls'] },
+    rotationEnabled: { default: true },
+    rotationSensitivity: { default: 0.05 } // radians/frame, ish
```

Comment:
Maybe. In this case there's an overall sensitivity, and mouse-controls has a property to override it. Could move sensitivity directly into `gamepad-controls` and `mouse-controls` since it doesn't apply to `hmd-controls` anyway.

Easing/acceleration make more sense at the top level to me, since movement speed should probably be consistent.